lib/rubocop/cop/style/accessor_grouping.rb

```diff
@@ -7,6 +7,9 @@ module Style
       # By default it enforces accessors to be placed in grouped declarations,
       # but it can be configured to enforce separating them in multiple declarations.
       #
+      # Note: `Sorbet` is not compatible with "grouped" style. Consider "separated" style
```

Comment:
I don't think AsciiDoc handles admonition markers properly if they are not UPCASE like NOTE.

Reply:
Fixed, thanks for letting me know.